tests/lib/rules/no-send.js

```diff
@@ -0,0 +1,46 @@
+const rule = require('../../../lib/rules/no-send');
+const MESSAGE = rule.meta.message;
+const RuleTester = require('eslint').RuleTester;
+const ruleTester = new RuleTester();
+
+ruleTester.run('no-send', rule, {
+  valid: [
+    {
+      code: `
+        function anotherFunction() {
+        }
+
+        export default Ember.Component({
+          actions: {
+            userInput() {
+              anotherFunction();
+              this.myAction();
```

Comment:
`someObj.send();` will fail, but should not

Reply:
@nlfurniss I've modified to check for calls of `send` on `this` but now that I think about it that might not be the best approach. 

We basically want to do this check on objects that use the `ActionHandler` mixin. 

Did you have anything in mind to prevent the rule to fail with `someObj.send()`?